students/mimalvarez_pintor/lesson02/Grid_Printer.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 13 21:42:59 2020
+
+@author: miriam
+"""
+# Printing a grid part 1
+n= 1
+m= 4
+
+plus= '+'
+minus= ' -'
+wall='|'
+space= '  '
+def line():
+    print(n*plus,m*minus,n*plus, m*minus,n*plus)
+def other():
+    print(n*wall,m*space,n*wall,m*space,n*wall)
+    print(n*wall, m*space, n*wall, m*space, n*wall)
+    print(n*wall, m*space, n*wall, m*space, n*wall)
+    print(n*wall, m*space, n*wall, m*space, n*wall)
+line()
+other()
+line()
+other()
+line()
+
+# Printing a grid part 2
+plus= '+'
+minus= '-'
+wall='|'
+space= ' '
+def print_grid(l):
+    Newl=l+2
+    for j in range(Newl):
+        if j == 0 or j == int(Newl/2) or j == Newl -1:
+            for i in range(Newl):
+                if i == 0 or i == int(Newl/2):
+                    print (plus, end = space)
```

Comment:
No spaces after function names:  print(
Also, Within these parentheses you don't want spaces around the assignment operator:  print(plus, end=space).